aisdc/generate_report.py

```diff
@@ -291,22 +291,10 @@ def process_json(input_filename: str, output_filename: str):
 
     modules = [
         FinalRecommendationModule(json_report),
-        SummariseUnivariateMetricsModule(json_report),
-        SummariseAUCPvalsModule(json_report, p_thresh=0.05),
```

Comment:
Maybe we can quickly discuss -- I think there is a risk that in this first example, if we simplify things too much we paint a picture for TREs that things are easy...will discuss. Appreciate that we need to get it finalised now.

Reply:
Actually, ignore this. No time at the moment. However, I'll make an issue that we should refactor a bit so that your module re-uses code from some of the others.